main.css

```diff
@@ -344,3 +346,356 @@
     color: black;
   }
 }
+
+@media only screen and (min-width: 768px){
+  body{
+    color: #091E42;
+  }
+
+  .header-section {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items:center;
+    background-color: #3e3c3c;
+    color: #fff;
+    padding:10px;
+    width: 100%;
+  }
+
+  .header-section .title {
+    text-decoration: none;
+    color: #fff;
+    margin-left: 20px;
+  }
+
+  .header-section .menu {
+    display: flex;
+    flex-direction: row;
+    margin-right: 20px;
+  }
+
+  .header-section .menu .menu-links {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .header-section .menu .menu-links a {
+    text-decoration: none;
+    list-style: none;
+    margin-right: 14px;
+    color: white;
+  }
+
+  .header-section .toggle-icon {
+    display: none;
+  }
+
+  .hero-section {
+    background: #1c1a19 url(./images/heroDesktopBg.png);
+    background-repeat: no-repeat;
+    background-size: 380px;
+    background-position-x: 930px;
+    background-position-y: -44px;
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: flex-end;
+    width: 100%;
+    height: 440px;
+    gap: 50px; 
+  }
+
+  .hero-section .hero-text {
+    color: #fff;
+    width: 400px;
+    height: 123px;
+  }
+  
+  .hero-section .hero-title h1 {
+    color: #f55800;
+    font-size: 72px;
+    font-family: 'Crete regular', serif;
+  }
+
+  .hero-section .hero-title h3 {
+    font-family: 'Inter', sans-serif;
+  }
+
+  .hero-section .hero-text .hero-desc {
+    font-family: 'Inter regular', sans-serif;
+    padding-top: 12px;
+    line-height: 20px;
+    font-size: 12px;
+  }
+
+  .hero-section .hero-social-links {
+    margin-left:16px;
+    list-style: none;
+    padding-top: 100px;
+  }
+
+  .hero-section .hero-social-links img {
+    width: 60%;
+    margin-top: 10px;
+  }
+
+  .works-section .Work-section-title {
+    display: flex;
+    flex-direction: row;
+    gap: 30px;
+    margin: 50px 50px 50px 50px;
+  }
+
+  .works-section .Work-section-title .vector {
+    width: 65%;
+    height: 3px;
+    margin-top: 12px;
+  }
+
+  .works-section .Work-section-title h2{
+    color: #091E42;
+  } 
+
+  .works-section .works-text {
+    display: flex;
+    flex-direction: row;
+    gap: 14px;
+    margin: 0 50px 50px 50px;
+  }
+
```

Comment:
- [ ] Hey, you're doing great with your design, however, you need to add this adjustment.
The image with the `.placholder` class is big and takes up 60% of the width in your `.works-text` container and the rest content takes up the remaining. 
Right now, your content is taking the width of the image. Please adjust your styling to fix this.